scripts/create_channel.go

```diff
@@ -0,0 +1,73 @@
+package scripts
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"go.uber.org/zap"
+
+	"github.com/Brightscout/mattermost-load-test-scripts/serializers"
+	"github.com/Brightscout/mattermost-load-test-scripts/utils"
+)
+
+func CreateChannels(config *serializers.Config, logger *zap.Logger) error {
+	connectionConfiguration := config.ConnectionConfiguration
+	client := model.NewAPIv4Client(connectionConfiguration.ServerURL)
+	if _, _, err := client.Login(connectionConfiguration.AdminEmail, connectionConfiguration.AdminPassword); err != nil {
+		return err
+	}
+
+	var newChannels []*serializers.ChannelResponse
+	response, err := utils.LoadResponse()
+	if err != nil {
+		return err
+	}
+
+	for _, channel := range config.ChannelsConfiguration {
+		team, _, err := client.GetTeamByName(channel.MMTeamName, "")
```

Comment:
We should create functions like `CreateOrGetTeam` and `CreateOrGetChannel` and use those. Those functions will try to get the respective team/channel and return them if found. If not, they will create the respective team/channel and return them.